### PR TITLE
Release branch for v2.2.1

### PR DESCRIPTION
**Pull request by Kami**
Release branch for v2.2.1 which includes the following changes:

* Include st2tests Python component in final st2 package distribution - #438

### CODE REVIEW

**Comment by Kami** [comment]
@armab If there are any changes you would like to include in v2.2.1, please cherry-pick it into this branch.

**Comment by Kami** [comment]
It looks like tests are failing because it's using old v2.2 st2 code - they should pass once we merge v2.2.1_changes branch in st2 into v2.2 branch.

**Comment by Kami** [comment]
@enykeev^ I also noticed some of the docker-compose changes you made in master haven't been merged into v2.2 branch yet.

I assume we also want those changes in v2.2 branch?

**Comment by LindsayHill** [comment]
@kami yeah, probably do. Funny thing is that once we release 2.2.1 I think the original version matching problem will be hidden again

**Comment by armab** [comment]
Thanks @Kami, I've included my changes.

File: scripts/st2bootstrap-deb.sh [hunk 1]
```diff
@@ -552,9 +555,11 @@ configure_st2chatops() {
 verify_st2() {
 
   # TODO: This is a temporary and nasty workaround for xenial CI failures.
+  # TODO: Fix https://github.com/StackStorm/st2/issues/3290
```
Comment (armab) [inline scripts/st2bootstrap-deb.sh]: @Kami I hope to get your help with this Issue to get rid of nasty sleeps.

**Comment by Kami** [comment]
OK, merged v2.2.1_changes branch in v2.2 in st2 and tests are passing now.

Will also go ahead and merge this branch.

**Pull request merged**